oak_restricted_kernel_sdk/src/dice.rs

```diff
@@ -35,11 +35,17 @@ use zerocopy::{AsBytes, FromZeroes};
 pub struct DiceWrapper {
     pub evidence: Evidence,
     pub encryption_key: EncryptionKeyProvider,
-    pub signing_key: SigningKey,
+    pub signer: SigningOracle,
+}
+
+impl DiceWrapper {
```

Comment:
I don't think the work DICE should be a part of our SDK, because in the future we might want to switch to a different attestation mechanism (i.e. RTMR)

Reply:
Discussed offline with @andrisaar & @ipetr0v, the consensus is that we will implement the somewhat magical solution with the `OnceCell` for now, and will remove it once we have an orchestrator on restricted kernel.

Reply:
I would still suggest merging this PR as is. I will implement the aforementioned magic in a subsequent PR. Exposing the `DiceWrapper` is not a new change introduced by this PR, so imo it's fine to keep for now.